pkg/controller/epgcache.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2021 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import (
+	"github.com/noironetworks/aci-containers/pkg/apicapi"
+	"github.com/noironetworks/aci-containers/pkg/metadata"
+
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+)
+
+func (cont *AciController) clearFaultInstances() {
+	args := []string{
+		"order-by=vmmClusterFaultInfo.modTs|desc",
+	}
+	faultUri := fmt.Sprintf("/api/node/class/vmmClusterFaultInfo.json?%s", strings.Join(args, "&"))
```

Comment:
Is there a way to query just for vmmClusterFaultInfo instances under this VMM instead of retrieving all and skipping the ones for other VMMs?

Reply:
Yes this is achievable by applying the query filters. I have incorporated this change